src/main/java/seedu/address/model/resident/UniqueResidentList.java

```diff
@@ -86,11 +88,13 @@ public void remove(Resident toRemove) {
         if (!internalList.remove(toRemove)) {
             throw new ResidentNotFoundException();
         }
+        FXCollections.sort(internalList);
```

Comment:
Removing probably no need to sort again but I think it is ok to have it here too since it probably does not affect performance much at this scale